doc/guide/https.xml

```diff
@@ -30,11 +30,16 @@
 
     <para>Cockpit will load a certificate from the <code>/etc/cockpit/ws-certs.d</code>
       directory. It will use the last file with a <code>.cert</code> extension in
-      alphabetical order. The <literal>.cert</literal> file should contain at least two
-      OpenSSL style PEM blocks. First one or more <literal>BEGIN CERTIFICATE</literal>
-      blocks for the server certificate and the intermediate certificate authorities
-      and a last one containing a <literal>BEGIN PRIVATE KEY</literal> or similar.
-      The key may not be encrypted. For example:</para>
+      alphabetical order. The <literal>.cert</literal> file should contain
+      one or more OpenSSL style <literal>BEGIN CERTIFICATE</literal>
+      blocks for the server certificate and the intermediate certificate authorities.</para>
+
+    <para>The private key can either be contained in the same <code>.cert</code> file
```

Comment:
Do we want to soft-deprecate the merged file functionality?  I always found it to be very strange, to be honest...

Reply:
We can consider that, and I'd be fine with that. Then remotectl should create the self-signed or sscg certificates in split format, too, and the doc should be adjusted. This sounds like a separate PR, though.